src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.Scanner;
+
 public class Duke {
+    static final String line_separator = "     _______________________\n";
+    static final String spacing = "     ";
```

Comment:
Constants naming should be all in uppercase with underscores, otherwise good job on using constants to make the codes look neater